api/src/api/organizations_v1/organization_routes.py

```diff
@@ -156,3 +161,33 @@ def organization_update_user_roles(
         )
 
     return response.ApiResponse(message="Success", data=roles)
+
+
+@organization_blueprint.post("/<uuid:organization_id>/invitations")
+@organization_blueprint.input(OrganizationCreateInvitationRequestSchema, location="json")
+@organization_blueprint.output(OrganizationCreateInvitationResponseSchema)
+@organization_blueprint.doc(responses=[200, 400, 401, 403, 404, 422])
+@organization_blueprint.auth_required(api_jwt_auth)
+@flask_db.with_db_session()
+def organization_create_invitation(
+    db_session: db.Session, organization_id: UUID, json_data: dict
+) -> response.ApiResponse:
+    """Create an invitation for a new organization member"""
+    add_extra_data_to_current_request_logs(
+        {"organization_id": organization_id, "invitee_email": json_data.get("invitee_email")}
```

Comment:
Don't add email to this, this adds it to all of our logs, we don't want emails in our logs.
```suggestion
        {"organization_id": organization_id}
```